docs/source/reference/functions.rst

```diff
@@ -302,6 +302,7 @@ Normalization functions
    chainer.functions.layer_normalization
    chainer.functions.local_response_normalization
    chainer.functions.normalize
+   chainer.functions.standardize
```

Comment:
If I understand https://github.com/chainer/chainer/pull/6678#discussion_r270714744 correctly, this would be either `chainer.functions._standardize` or removed.

Reply:
ah! I've forgotten. thank you.